### PR TITLE
Michael haberman talks proposal

### PR DESCRIPTION
**Pull request by habmic**

### CODE REVIEW

**Comment by habmic** [comment]
Hope you like them :-)

**Comment by urish** [comment]
Thanks @habmic  !

File: angular-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Session Abstract
+
+In this talk we will go over some of the cool tools out there to help us better develop our Angular application, we will also look at some libraries to support angular and some modules to use to make out development process easier. 
+ 
+### Session Meta Data
+
+- Target Audience: Beginners
+- Length: Full session (45 minutes)
+- Tags: Tools,Angular
```
Comment (olgakogan) [inline angular-tools/README.md]: can you please add the talk duration?

File: angular-unit-test/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+what it was designed to do.  Recently web application developers are also
+seeking for testing option as their applications are increasing and getting
+more complex. In this session we will review how to test Angular application in
+Karma and Jsamine.
+
+### Session Meta Data
+
+- Target Audience: Beginners
+- Length: Full session (45 minutes)
+- Tags: Unit-testing,Angular
```
Comment (olgakogan) [inline angular-unit-test/README.md]: can you please add the talk duration?

Comment (habmic) [inline angular-unit-test/README.md]: 45 min

File: angular-tools/README.md (hunk 1 above)
Comment (habmic) [inline angular-tools/README.md]: 45 min

**Comment by olgakogan** [comment]
:+1:

**Pull request merged**